aten/src/ATen/native/DistributionTemplates.h

```diff
@@ -77,6 +82,16 @@ static void check_from_to_in_range(int64_t from, int64_t to_inc, caffe2::TypeMet
       const auto max = static_cast<double>(std::numeric_limits<scalar_t>::max());
       CHECK_OUT_OF_BOUNDS(from, "from", min, max, dtype);
       CHECK_OUT_OF_BOUNDS(to_inc, "to - 1", min, max, dtype);
+
+      constexpr auto digits = std::numeric_limits<scalar_t>::digits;
+      const auto two_pow_digits = 1LL << digits;
+      const auto minus_two_pow_digits = -two_pow_digits;
+      const auto two_pow_digits_name = "2^" + std::to_string(digits);
+      const auto minus_two_pow_digits_name = "-(2^" + std::to_string(digits) + ")";
```

Comment:
Can all of these be made constexpr? If not, it seems like it would be better to ensure that we only render the strings inside of the failing if statement

Reply:
addressed